Gopkg.toml

```
@@ -28,4 +28,7 @@
 
 [[override]]
   name = "github.com/spf13/cobra"
-  revision = "7547e83b2d85fd1893c7d76916f67689d761fecb"
\ No newline at end of file
+  revision = "7547e83b2d85fd1893c7d76916f67689d761fecb"
+[[constraint]]
+  branch = "master"
```

```suggestion
  revision = "89fcab3d43de07060e4fd4c1547430ed57e87f24"
```

Use `revision` instead of `branch`. The value of the revision is the current commit hash of the master branch. In this way, we will not be affected when new commits are contributed to the master branch